plugins/notification-resources/src/components/EmployeeInbox.svelte

```diff
@@ -76,22 +57,41 @@
     }
   )
 
+  $: updateDocs(docs, accountId)
+  function updateDocs(docs: DocUpdates[], accountId: Ref<Account>, forceUpdate = false) {
+    if (loading && !forceUpdate) {
+      return
+    }
+
+    _docs = []
```

Comment:
Better to collect into local variable, to prevent flicking.

Reply:
Done, thanks